### PR TITLE
Open room members when member info label clicked

### PR DESCRIPTION
**Pull request by LorenDB**

### CODE REVIEW

File: src/timeline/TimelineModel.cpp [hunk 1]
```diff
@@ -1067,9 +1067,9 @@ TimelineModel::openUserProfile(QString userid)
 }
 
 void
-TimelineModel::openRoomMembers()
+TimelineModel::openRoomMembers(QString room_id)
 {
-        MemberList *memberList = new MemberList(roomId());
+        MemberList *memberList = new MemberList(room_id == QString() ? roomId() : room_id, this);
```
Comment (deepbluev7) [inline src/timeline/TimelineModel.cpp:1072]: Can you change that to always require an explicit room id?

Comment (LorenDB) [reply src/timeline/TimelineModel.cpp:1072]: Actually, I've been considering moving some of this room id-dependent stuff into TimelineViewManager. I'll probably do that instead

Comment (deepbluev7) [reply src/timeline/TimelineModel.cpp:1072]: The problem here is that I am considering having multiple rooms open at once. So everything should have the memberlist from the current room. In which case roomId() is probably right in this case and you shouldn't be able to pass a room id.

Comment (deepbluev7) [reply src/timeline/TimelineModel.cpp:1072]: (Or have a function in the timelinemanager, that can be passed a roomId, but not both)

**Pull request merged**